.eslintignore

```diff
@@ -1 +1,2 @@
-public/
\ No newline at end of file
+public/
+webpack.config.js
```

Comment:
Why should `webpack.config.js` be ignored? How can other developers build it locally?

Reply:
I just thought `wepack.config.js` should not be linted

Reply:
Ah, my bad. Read the filename as gitignore